tests/unit/Switch-bidi.js

```diff
@@ -0,0 +1,136 @@
+define([
+	"intern!object",
+	"intern/chai!assert"
+], function (registerSuite, assert) {
+	var container, html = "<d-switch id='b1' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' textDir='ltr'></d-switch>" +
+		"<d-switch id='b2' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='false' textDir='rtl'></d-switch>" +
+		"<d-switch id='b3' checkedLabel='\u05d0\u05d1\u05d2' uncheckedLabel='ABC' checked='true' title='ABC \u05d0\u05d1\u05d2' textDir='auto'></d-switch>";
+
+	registerSuite ({
+		name: "deliteful/Switch (bidi)",
+		setup: function () {
+			moduleRequire = require.config({
```

Comment:
Hmm, I forgot about that.  Your solution sounds fine, although I think another approach would be to have some kind of clear() method on delite/register.